src/js/components/TableCell/index.d.ts

```diff
@@ -7,6 +8,6 @@ export interface TableCellProps {
   verticalAlign?: "top" | "middle" | "bottom";
 }
 
-declare const TableCell: React.FC<TableCellProps & JSX.IntrinsicElements['td']>;
+declare const TableCell: TableCellProps & Box & JSX.IntrinsicElements['td'];
```

Comment:
what does it mean removing the wrapper of `React.FC<>`?

Reply:
React.fc is a short for a function component we will actually need this and have to export Box without the component class. Table cell would be a functional component where box would need to be a component class in which it already is. The class component which is box would be used just to return a component.